statviz/src/components/custom-graphs/BarChartVertical.tsx

```diff
@@ -0,0 +1,177 @@
+import { Grid } from "@visx/grid";
+import { labelProps, tickProps, tooltipStyles } from "../../utils/chart";
+import { Group } from "@visx/group";
+import { scaleLinear } from "@visx/scale";
+import { AxisBottom, AxisLeft } from "@visx/axis";
+import { scaleBand } from "d3-scale";
+import { Bar } from "@visx/shape";
+import { useTooltip, Tooltip } from "@visx/tooltip";
+import { localPoint } from "@visx/event";
+
+type TooltipData = string;
+
+const marginTop = 20;
```

Comment:
@MaikNeubert it feels weird to hard-code and define it here. Also is this 20 px or rem or ??

Reply:
BarChartVertical.tsx and PieChartOld.tsc in custom-graphs will be removed in one of the upcoming PRs as they will be replaced by nivo

Reply:
Every type of chart needs different margins depending on legends and other information that might be displayed for a certain chart type. So for me it makes sense to have it inside the chart component.
If it can be generalized i will put it somewhere more general in the future.

Reply:
also the margins here do not represent CSS style values but values used for rendering margins inside SVG where everything is in pixels